ui/pages/notifications/snap/types/types.ts

```diff
@@ -1,8 +1,15 @@
-export const SNAP = 'SNAP' as const;
+import { SNAP_TRIGGER } from '../../notification-components';
+
+export type SnapNotificationExpandedView = {
```

Comment:
Maybe this should be a type in `snaps-sdk`, so we can re-use it.

Reply:
I can open an issue to add it in after and replace this.